src/olympia/addons/tests/test_indexers.py

```diff
@@ -412,6 +412,10 @@ def test_extract_translations_engb_default(self):
     def test_extract_persona(self):
         # Override self.addon with a persona.
         self.addon = addon_factory(persona_id=42, type=amo.ADDON_PERSONA)
+        # Delete any files so that we can be sure that compat info is entirely
+        # done automatically as long as it's a persona.
+        self.addon.current_version.files.all().delete()
```

Comment:
I thought you fixed `addon_factory` to not add files for persona type addons?

Reply:
I did but I actually wrote this code first (that's what made me realize `addon_factory()` was doing that) and forgot to patch it now that the fix has been merged. I'll remove this line (or better yet, replace it with an `assert` making sure there are no files)